helm_deploy/values-preprod.yaml

```diff
@@ -14,5 +14,9 @@ generic-service:
     GA_ID: G-08D8T0RZZR
     FEATURE_SP_REPORTING_ENABLED: true
     SP_PERCENTAGE_OF_DASHBOARD_PAGINATION_USERS: 100
+    SP_MY_CASES_PAGE_SIZE: 10
+    SP_OPEN_CASES_PAGE_SIZE: 10
+    SP_UNASSIGNED_CASES_PAGE_SIZE: 10
+    SP_COMPLETED_PAGE_SIZE: 10
```

Comment:
_nit_ Did you mean to put this in dev? In preprod, we usually want to replicate the traffic/volumes/load as on production

Reply:
Dev already the same config and I thought keeping the same thing in pre-prod. In prod, the config is